sources/packages/backend/libs/sims-db/src/entities/cas-supplier.model.ts

```diff
@@ -1,7 +1,14 @@
-import { TableNames } from "@sims/sims-db/constant";
+import { ColumnNames, TableNames } from "@sims/sims-db/constant";
 import { RecordDataModel } from "@sims/sims-db/entities/record.model";
+import { Student } from "@sims/sims-db/entities/student.model";
```

Comment:
The index can be used instead as `import { Student } from "@sims/sims-db/entities";`.